grant-project/grant-monorepo
# Profile UI

Closes #120.

### What this does
- provides /profile/<ethAddress> route with user Profile information
- creates `modules/users`
- extends `TeamMember` type
- expects future api support for fetching user created & funded proposals as well as user comments
- mocks out a few comments for each proposal
- mocks `proposal.funded` and `proposal.target`, I figure this can be hooked-up post server web3 read enhancement
- currently just returns all proposals from the api for both funded and created

### Test
- visit a proposal detail and click on a team member to be taken to their profile
<img width="328" alt="screen shot 2018-10-01 at 10 16 18 pm" src="https://user-images.githubusercontent.com/11430954/46327578-e8d16180-c5c7-11e8-9c26-1671a8be39ae.png">

- explore funded and created proposals, check that team member links and proposal links work
<img width="808" alt="screen shot 2018-10-01 at 10 19 12 pm" src="https://user-images.githubusercontent.com/11430954/46327600-14ece280-c5c8-11e8-8441-8de181a7e4a0.png">

- click profile icon in upper-right, you should be taken to auth'd user profile
<img width="245" alt="screen shot 2018-10-01 at 10 16 54 pm" src="https://user-images.githubusercontent.com/11430954/46327582-eff86f80-c5c7-11e8-89c1-b89691c9f0ad.png">
